.github/workflows/pull-components.yml

```diff
@@ -35,15 +41,21 @@ jobs:
+            {order: 7 , repo : "ishitatsuyuki/LatencyFleX"       , workflow: "artifact.yml"           , branch: "master", name-prefix: "latencyflex-" , name-suffix: ""                  , version-prefix: ""         , Category: "latencyflex", Sub-category: ""      , yaml-file: "17-latencyflex.yml"  },
+            {order: 8 , repo : "jp7677/dxvk-nvapi"               , workflow: "artifacts.yml"          , branch: "master", name-prefix: "dxvk-nvapi-"  , name-suffix: ""                  , version-prefix: ""         , Category: "nvapi"      , Sub-category: ""      , yaml-file: "16-dxvk-nvapi.yml"   },
+            {order: 9 , repo : "Sporif/dxvk-async"               , workflow: ""                       , branch: "master", name-prefix: "dxvk-async-"  , name-suffix: ""                  , version-prefix: ""         , Category: "dxvk"       , Sub-category: ""      , yaml-file: "15-dxvk-async.yml"   },
+            {order: 10, repo : "Kron4ek/Wine-Builds"             , workflow: ""                       , branch: "master", name-prefix: "wine-"        , name-suffix: "-x86"              , version-prefix: ""         , Category: "runners"    , Sub-category: "wine"  , yaml-file: "11-kron4ek.yml"      },
+            {order: 11, repo : "Kron4ek/Wine-Builds"             , workflow: ""                       , branch: "master", name-prefix: "wine-"        , name-suffix: "-staging-x86"      , version-prefix: ""         , Category: "runners"    , Sub-category: "wine"  , yaml-file: "11-kron4ek.yml"      },
+            {order: 12, repo : "Kron4ek/Wine-Builds"             , workflow: ""                       , branch: "master", name-prefix: "wine-"        , name-suffix: "-staging-tkg-x86"  , version-prefix: ""         , Category: "runners"    , Sub-category: "wine"  , yaml-file: "11-kron4ek.yml"      },
+            {order: 13, repo : "Kron4ek/Wine-Builds"             , workflow: ""                       , branch: "master", name-prefix: "wine-"        , name-suffix: "-staging-tkg-amd64", version-prefix: ""         , Category: "runners"    , Sub-category: "wine"  , yaml-file: "11-kron4ek.yml"      },
+            {order: 14, repo : "Kron4ek/Wine-Builds"             , workflow: ""                       , branch: "master", name-prefix: "wine-"        , name-suffix: "-staging-amd64"    , version-prefix: ""         , Category: "runners"    , Sub-category: "wine"  , yaml-file: "11-kron4ek.yml"      },
+            {order: 15, repo : "Kron4ek/Wine-Builds"             , workflow: ""                       , branch: "master", name-prefix: "wine-"        , name-suffix: "-amd64"            , version-prefix: ""         , Category: "runners"    , Sub-category: "wine"  , yaml-file: "11-kron4ek.yml"      },
```

Comment:
Kron4ek also builds stable versions of Valve's upstream Proton. Proton releases way less often, so they are harder to find, but if you look closely you'll see that #227 adds two builds of Proton: `kron4ek-wine-7.0-5-proton-amd64` and `kron4ek-wine-7.0-6-proton-amd64`.

Reply:
Oh, I definitely missed those builds. However, even with their misleading name, they're Wine builds and not Proton builds: `Note that DXVK and vkd3d-proton are not included in my wine-proton builds`.

The fact that this bug went undiscovered until now tells me that those wine-proton variants are used by absolutely no one. I should nonetheless be able to add them without requiring additional hacks/tweaks. I'm leaving this review unresolved until then.

Reply:
What bug? The previous versions were added as proton builds before. They are hard to find on the UI, because they currently show up at the bottom of the Proton-GE list, but they are there, people can use them.

Reply:
The bug is that `Sub-category` should be `wine` for those builds, not `proton`. When using a Proton variant, the Steam runtime (Soldier) is automatically enabled by Bottles, that won't work with regular Wine runners (could lead to crash). Proton runners also include `vkd3d-proton` and `dxvk` on their own, whereas Wine runners do not.

Reply:
I greatly underestimated what would be required to support this new wine-proton builds. As is, there's not much I can do. I'll open an issue on the Kron4ek repository gently asking if the naming convention could be change for these builds. Closing for now.

Reply:
I don't get it, what's the problem with the name of the packages? Changing the sub-category here shouldn't require any changes from upstream.

Reply:
Indeed, those are two separate issues.

The problem with the `wine-proton` naming is that it currently cannot fit into the expected `name-prefix`, `name-suffix` and `version-prefix` format which describes each component. Or more precisely it can, but would clash with the format of the regular `wine`, ending up in an impossible situation.

Reply:
Seems like just `name-prefix: "wine-"        , name-suffix: "-proton-amd64"` to me. Anyway, let's make sure to not bother upstream with what's clearly downstream CI problems, it's not helpful. If the problem can not be solved declaratively in yaml, then it can be solved imperatively somewhere else. That shouldn't be hard.

Reply:
Kron4ek accepted to change the `proton-wine` naming scheme for future releases. They will now work flawlessly.